publication-rest/src/test/java/no/unit/nva/publication/delete/DeletePublicationHandlerTest.java

```diff
@@ -286,6 +301,8 @@ void shouldSuccessWhenUserIsContributorWhenUnpublishingPublication()
 
         var response = GatewayResponse.fromOutputStream(outputStream, Void.class);
         assertThat(response.getStatusCode(), is(equalTo(SC_ACCEPTED)));
+
+        verify(eventBridgeClient, Mockito.times(1)).putEvents(argThat(new EventBridgeRequestMatcher()));
```

Comment:
I think I would assert this explicitly in another test: `shouldProduceUpdateDoiEventWhenDeletionIsSuccessful()`, and assert that the PutEventsRequestEntry contains the expected detail (DoiMetadataUpdateEvent)